libs/model/src/community/GetTransactions.query.ts

```diff
@@ -0,0 +1,80 @@
+import type { Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { QueryTypes } from 'sequelize';
+import { models } from '../database';
+
+export function GetTransactions(): Query<typeof schemas.GetTransactions> {
+  return {
+    ...schemas.GetTransactions,
+    auth: [],
+    secure: false,
```

Comment:
Should we secure this? At least to check if the user is authenticated?

Reply:
It was required to be viewable in the public user profile so I left it like this.